.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: Bug Report
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
```

Comment:
The description of a section could go in a comment tag. Like that it didn't display on rendering.

```suggestion
<!--- A clear and concise description of what the bug is. -->
```

Reply:
Done as desired